parameters.yml.dist

```diff
@@ -15,6 +15,14 @@ forwarded_ports: {
     "3000": "3000",  # BrowserSync default port
 }
 
+# Virtual machine memory size use in MB. Defaults to 4096. Uncomment and change this
```

Comment:
Can you please add a note that these settings are only used with the VirtualBox provider?